include/swift/Demangling/Demangle.h

```diff
@@ -67,6 +68,7 @@ struct DemangleOptions {
     Opt.ShortenThunk = true;
     Opt.ShortenValueWitness = true;
     Opt.ShortenArchetype = true;
+    Opt.ShowLocalDiscriminators = true;
```

Comment:
Shouldn't this one be `false`, since ShowPrivateDiscriminators and DisplayModuleNames are also `false`?

Reply:
[This](https://github.com/apple/swift/blob/master/test/SILGen/generic_local_property.swift#L24) check fails if it's `false`. We could customize options [here](https://github.com/apple/swift/blob/master/lib/SIL/SILPrinter.cpp#L74), to address that case.

Reply:
I think I would just pass `-Xllvm -sil-full-demangle` for that one test.